### PR TITLE
Fix non fp v

### PR DESCRIPTION
**Pull request by Avimitin**
Continue works on #53. Waiting for review on macro modification https://github.com/riscv/riscv-test-env/pull/45.

### CODE REVIEW

File: generator/insn.go [hunk 1]
```diff
@@ -296,6 +298,17 @@ func (i *Insn) genHeader() string {
 
 RVTEST_RV%dUV
 `, i.Name, i.Option.XLEN)
```
Comment (ksco) [inline generator/insn.go:300]: Suggested changes:
```go
RVTEST_RV%dUV%s
`, i.Name, i.Option.XLEN, iff(i.Option.Fp, "", "X"))
```

**Review by ksco — COMMENTED** [review]
Thanks! Overall looks good, a minor change is proposed.

**Comment by Avimitin** [comment]
Fixed. Waiting for CI pass.

**Comment by ksco** [comment]
> Fixed. Waiting for CI pass.

It's a capital X.

**Comment by Avimitin** [comment]
Oops...It should be correct now.

**Pull request merged**